### PR TITLE
Transfered potatoe recipe into the required structure

### LINKED ISSUE #40: New patatoe recipe
Hey! You could enjoy a potatoe recipe, could you add it please? Thanks!

### PR DESCRIPTION
**Pull request by Mirarufeger**
Thanks for the review, this in the potatoe recipe in the updated structure. See last commit :)

fixes #40

### CODE REVIEW

**Review by lnnrtwttkhn — CHANGES_REQUESTED** [review]
Hi @Mirarufeger! Thank for this PR! I'm a bit confused, since there are two files in this PR. Could you please delete the one that is not needed? Thanks!

**Comment by Mirarufeger** [comment]
Hi! I deleted the unnecessary file, everything should be alright now. :)

File: Potatoe recipe [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Very Crispy Potatoes"
+image: "https://emmikochteinfach.de/wp-content/uploads/2022/11/Ablauf-Knusprige-Ofenkartoffeln-%E2%80%93-Kartoffel-Wedges-10.jpg"
+image-alt: "https://emmikochteinfach.de/wp-content/uploads/2022/11/Ablauf-Knusprige-Ofenkartoffeln-%E2%80%93-Kartoffel-Wedges-1.jpg"
+categories:
+  - salty
+  - savoury 
+  - vegan 
+  - carbs
```
Comment (lnnrtwttkhn) [inline Potatoe recipe:9]: Would you like to add your author information? Add it to the YAML header like this:

```YAML
author: firstname lastname
```

**Review by lnnrtwttkhn — CHANGES_REQUESTED** [review]
Thanks for this PR @Mirarufeger! Would you like to add your author information? See comment above.

Also, please move the file to the `/recipes` subfolder and rename to `potatoes.qmd`.

You can do this from the command line, executed from the root of the directory, like this:

```bash
mv "Potatoe recipes.qmd" recipes/ppotatoes.qmd
```

Thanks!

**Comment by lnnrtwttkhn** [comment]
> Also, please move the file to the /recipes subfolder and rename to potatoes.qmd.

I just added this in https://github.com/lnnrtwttkhn/recipes/pull/63/commits/c1c6d0ed74807d8d384eb32b9c521b33d31c8d0f.

@Mirarufeger feel free to send another PR, if you still want to add your author information.

Ready to merge now! 🚀

**Comment by lnnrtwttkhn** [comment]
@allcontributors please add @Mirarufeger  for bug, content, code

**Pull request merged**